techniques/css/C42.html

```diff
@@ -135,14 +133,14 @@ <h4>The CSS</h4>
 			
 			</section>
 			<section class="example" id="example-2-using-a-display-value-of-flex-and-min-height-/-width">
-				<h3>Example 2: Using a <code class="language-css">display</code> value of <code class="language-css">flex</code> and <code class="language-css">min-height</code> / <code class="language-css">width</code></h3>
-				<p>The second example uses <code class="language-css">min-width</code> and <code class="language-css">min-height</code> on the targets (the linked numbers in the pagination menu) and not on the parent container, thereby meeting this target spacing success criterion and incidentally also the AAA Success Criterion 2.5.5 Target Size.</p>
+				<h3>事例 2: <code class="language-css">display</code> 値に <code class="language-css">flex</code> 及び <code class="language-css">min-height</code> / <code class="language-css">width</code> を使用する</h3>
+				<p>2 番目の事例では、親コンテナではなくターゲット (ページネーションメニュー内のリンクされた数字) に <code class="language-css">min-width</code> 及び <code class="language-css">min-height</code>  を使用しているため、このターゲットの間隔の達成基準を満たし、さらに AAA 達成基準 2.5.5 ターゲットサイズも満たしている。</p>
```

Comment:
incidentallyは「偶然」「たまたま」「ついでに」という感じだと思いますが、まあいいか……。

linked numbers の訳は前と揃えて:

```suggestion
				<p>2 番目の事例では、親コンテナではなくターゲット (ページネーションメニュー内のリンクになっている数字) に <code class="language-css">min-width</code> 及び <code class="language-css">min-height</code>  を使用しているため、このターゲットの間隔の達成基準を満たし、さらに AAA 達成基準 2.5.5 ターゲットサイズも満たしている。</p>
```

Reply:
> incidentallyは「偶然」「たまたま」「ついでに」という感じだと思いますが、まあいいか……。

であれば、こうするのはどうでしょう？

```suggestion
				<p>2 番目の事例では、親コンテナではなくターゲット (ページネーションメニュー内のリンクになっている数字) に <code class="language-css">min-width</code> 及び <code class="language-css">min-height</code>  を使用しているため、このターゲットの間隔の達成基準を満たし、付随して AAA 達成基準 2.5.5 ターゲットサイズも満たしている。</p>
```